tests/console/yast2_bootloader.pm

```diff
@@ -27,6 +26,7 @@ sub run() {
     script_run("/sbin/yast2 bootloader; echo yast2-bootloader-status-\$? > /dev/$serialdev", 0);
     assert_screen "test-yast2_bootloader-1", 300;
     send_key "alt-o";                                     # OK => Close
+    if (assert_screen 'yast2_bootloader-missing_package') { send_key 'alt-i'; }
```

Comment:
that won't work. "if (assert_screen…" will not work as intended. Either the screen is shown, means the assert_screen returns successfully or the test will die because assert_screen throws an exception